Dockerfile

```diff
@@ -27,7 +27,7 @@ ENV DLANG_VERSION=$DLANG_VERSION
 ENV DLANG_EXEC=$DLANG_EXEC
 
 # Download and run the install script
-RUN curl -fsS -o /tmp/install.sh https://dlang.org/install.sh
```

Comment:
Why is this necessary ?

Reply:
dlang/installer#483 added support the new nightlies but is not yet available from dlang.org.
I'll revert this change and see if it still passes with the old nightlies.

Reply:
Didn't work, will need to wait until the new version is published to the homepage...

Reply:
Deployed the install.sh to dlang.org

Reply:
Thanks!